index.js

```diff
@@ -76,7 +91,22 @@ const dostroy = (config) => async (req, res, next) => {
   const all = !config || Object.keys(config).length === 0
   const r = config && config.rudy ? config.rudy : RUDY_DEFAULT
   const rl = config && config.rateLimiting ? config.rateLimiting : RATELIMITING_DEFAULT
+  const dynamic = config && config.dynamicRateLimiting ? config.dynamicRateLimiting : USE_DYNAMIC_RATE_LIMITING_DEFAULT
+  const userActiveTimeout = dynamic && config.userActiveTimeout && !isNaN(config.userActiveTimeout) ? config.userActiveTimeout : USER_ACTIVE_TIMEOUT_DEFAULT
```

Comment:
If config === undefined this will throw exception. consider: `dynamic && config && !isNaN(config.userActiveTimeout)`

Reply:
isNaN handles undefined values

Reply:
Yes, this is true. If config is null this will crash :O